src/sql_series.ts

```diff
@@ -2,6 +2,7 @@ import _ from 'lodash';
 
 export default class SqlSeries {
     series: any;
+    keys: any;
```

Comment:
Class `SqlSeries` also used in tests where it used within method `query` where `keys` are generated. To avoid tests failing we must add default `keys` value for `SqlSeries` objects [here](https://github.com/Vertamedia/clickhouse-grafana/blob/master/specs/datasource_specs.ts#L40) and [here](https://github.com/Vertamedia/clickhouse-grafana/blob/master/specs/datasource_specs.ts#L91).
Also it would be great if tests-specs will be populated with new tests.
I'm using `karma` to run tests:
`karma start -single-run`